### PR TITLE
Add Heli Pilot Coveralls retextures

### PR DESCRIPTION
**Pull request by JoramD0**
**When merged this pull request will:**
- Title

### CODE REVIEW

**Review by jonpas — CHANGES_REQUESTED** [review]
4 unused files?
- `coveralls_Theseus_black_co.png`
- `coveralls_Theseus_green_co.png`
- `coveralls_Theseus_grey_co.png`
- `coveralls_Theseus_khaki_co.png`

File: addons/units/config.cpp [hunk 1]
```diff
@@ -97,6 +97,10 @@ class CfgPatches {
             QCLASS(Item_Uniform_Garment_RS_TS_BP_BB),
             QCLASS(Item_Uniform_Garment_RS_TS_GP_BB),
             QCLASS(Item_Uniform_Garment_RS_TS_TP_TB),
+            QCLASS(Item_Uniform_HeliPilotCoveralls_Grey),
```
Comment (TyroneMF) [inline addons/units/config.cpp:100]:
```suggestion
            QCLASS(Item_Uniform_HeliPilotCoveralls_Black),
```

File: addons/units/config.cpp [hunk 2]
```diff
@@ -97,6 +97,10 @@ class CfgPatches {
...
+            QCLASS(Item_Uniform_HeliPilotCoveralls_Grey),
```
Comment (TyroneMF) [inline addons/units/config.cpp:101]:
```suggestion
            QCLASS(Item_Uniform_HeliPilotCoveralls_Green),
```

File: addons/units/config.cpp [hunk 3]
```diff
@@ -97,6 +97,10 @@ class CfgPatches {
...
+            QCLASS(Item_Uniform_HeliPilotCoveralls_Grey),
+            QCLASS(Item_Uniform_HeliPilotCoveralls_Grey),
```
Comment (TyroneMF) [inline addons/units/config.cpp:103]:
```suggestion
            QCLASS(Item_Uniform_HeliPilotCoveralls_Sand),
```

**Review by TyroneMF — CHANGES_REQUESTED** [review]

**Review by TyroneMF — APPROVED** [review]

**Review by jonpas — APPROVED** [review]

**Pull request merged**